chapters/modules/chapter_2/pages/calling_invoking.adoc

```diff
@@ -4,9 +4,111 @@
 
 Let's interact with our deployed voting contract.
 
-* Calling contracts refer to interacting with contract functions with the `@view` entry point (decorator). Since we are not altering the state of the network, and hence we do not need to pay fees, we do not need to sign the operation.
+* Calling contracts refer to interacting with contract functions with the `@view` entry point (decorator). Since we are not altering the state of the network, we do not need to pay fees and we do not need to sign the operation.
 * Invoking contracts is interacting with contract functions with the `@external` entry point. We need to sign the transaction and pay the operation fee.
 
+Here is the command that we used, to deploy the contract. Note that the voter accounts registered during deployment are the only ones that can vote.
+
+[source, bash]
+----
+starknet deploy --class_hash 0x3853a0a2697a834f808decbfcc01dc103d76bdcf2684975decf0793e4af896a --inputs 0x072c49c0f3a76640855c6d56104b034f2a61b95da5eb1c1a6afb8f7277856220 0x03de621c797439d2a5859c6cdebf3487ab84aa981e29a0e7fc0db455adab23d5 0x05c5b5acd90b327e45bf0076d52edfc4186d7e289c100625f630a53cee8cd57b --account vote_admin --max_fee 100000000000000000
+----
+
+== Checking for the ability to vote
+
+Before we start voting, we have the ability to check if a particular address/user can vote or not. There are two functions that are doing this check in the given contract: +
+
+[source,rust]
+----
+#[view]
+fn voter_can_vote(user_address: ContractAddress) -> bool {
+    // Read the voting status of the user from storage
+    can_vote::read(user_address)
+}
+----
+
+[source,rust]
+----
+#[view]
+fn is_voter_registered(address: ContractAddress) -> bool {
+    // Read the registration status of the address from storage
+    registered_voter::read(address)
+}
+----
+
+Both of the above functions are _view_ functions and therefore, we can use the _starknet call_ syntax to get the values from these functions. Let's use them alternatively to check the ability of the 3 voters to vote.
+
+You can directly pass the function name and the contract address in the _call_ syntax along with inputs(if any) to call a [#view] function. 
```

Comment:
Is using _call_ displayed nicely in antora?

Reply:
I have added a * around the Italics to make it look more prominent. Wdyt?